angular-workspace/projects/ni/nimble-angular/mapping/text/tests/nimble-mapping-text.directive.spec.ts

```diff
@@ -22,14 +22,14 @@ describe('NimbleMappingText', () => {
         @Component({
             template: `
                 <nimble-table>
-                    <nimble-table-column-enum-text key-type="boolean">
+                    <nimble-table-column-icon key-type="boolean">
```

Comment:
This is just transitional, since the plan is still to replace `nimble-table-column-icon` with `nimble-table-column-mapping`, right?

Reply:
That is correct. The `nimble-table-column-icon` will be renamed `nimble-table-column-mapping` -- probably in the next week or two.